### PR TITLE
Implemented Grape API with a Swagger frontend

### PR DESCRIPTION
**Pull request by dmlond**
/api_docs hosts an interactive swagger frontend instead of apipie frontend

### CODE REVIEW

File: app/api/aact2/v1/app_api.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+module AACT2
+  module V1
+    class AppAPI < Grape::API
+      desc 'app status' do
+        detail 'this returns a health status'
+        named 'app_storage'
+        failure [
```
Comment (garrettqmartin8) [inline app/api/aact2/v1/app_api.rb]: Not a big deal, but it's kinda weird that the failure clause would include the 200 status. I'm sure that's just a Grape/Swagger idiosyncrasy. Nothing to change though.

**Comment by garrettqmartin8** [comment]
Once those specs are fixed 👍  Thanks Darin!

File: app/api/aact2/v1/app_api.rb (hunk 1 above)
Comment (dmlond) [inline app/api/aact2/v1/app_api.rb]: this is a peculiarity with how grape_swagger generates documentation for the endpoint in swagger. It uses the failure array of arrays to generate all of the possible return codes. I figured this out when we did the Duke Data Service.  It generates the '[Response Messages](https://api.dataservice.duke.edu/apiexplorer#!/current_user/GET_api_version_current_user_format)' from it, which should include 200 (or 204, etc.).

**Comment by dmlond** [comment]
will check those specs and figure out where things are different from my local copy (all specs passed :P)

**Comment by dmlond** [comment]
hah! should have run rspec just one more time after the merge from official/master 😄 

should be fixed now.

**Comment by garrettqmartin8** [comment]
Great, thanks 👍

**Pull request merged**